src/components/internal/DataViewFilters.jsx

```diff
@@ -82,28 +122,33 @@ function DataViewFilters(props) {
 				}
 				{
 					optionalFilters.length > 0 &&
-					<Button
-						label={t("mosaic:DataView.more")}
-						variant="text"
-						color="black"
-						size="small"
-						iconPosition="right"
-						mIcon={ExpandMoreIcon}
-						mIconColor={theme.colors.gray600}
-						popover={
-							<PopoverDiv
-								className={`
-									${ props.loading ? "loading" : "" }
-								`}
-							>
-								<CheckboxList
-									options={options}
-									checked={activeFilters}
-									onChange={onActiveFiltersChange}
-								/>
-							</PopoverDiv>
-						}
-					/>
+					<>
+						<Button
+							label={t("mosaic:DataView.more")}
+							variant="text"
+							color="black"
+							size="small"
+							iconPosition="right"
+							mIcon={ExpandMoreIcon}
+							mIconColor={theme.colors.gray600}
+							onClick={onClick}						
+						/>
+						<DataViewFilterDropdown
+							anchorEl={state.anchorEl}
+							onClose={onClose}
+							onEntered={onEntered}
+							onExited={onExited}
+						>
+							<DataViewFilterMultiselectDropdownContent
+								comparison={''}

```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/simpleviewinc/sv-mosaic/pullRequest?prid=9715290)
```suggestion
								comparison={""}
```